dat/missions/zalek/neburesearch/neburesearch_00.lua

```diff
@@ -0,0 +1,221 @@
+mtitle = {}
+mtitle[1] = _("Novice Nebula Research")
+misn_reward = _("%s credits and the gratitude of a student")
+mdesc = {}
+mdesc[1] = _("Go to the %s system.")
+title = {}
+title[1] = _("Bar")
+title[2] = _("Please...")
+title[3] = _("What a mess...")
+title[4] = _("Ready")
+title[5] = _("Technical problems")
+title[6] = _("Shadows in the darkness")
+title[7] = _("Mission Success")
+text = {}
+text[1] = _([["Hello Captain! You are a pilot, right? For my project I require a ship heading to the Sol Nebula. Certainly you must be interested in the proposal of researching the phenomenon that cut us off from mankind's patrimony.
+    As this is the point where any other pilots I asked backed out I should start with my problem; due to some unfortunate circumstances the payment for this mission will be only 50000 credits. But rest assured, you will be mentioned in the acknowledgment section of my next paper!"]])
+text[2] = _([["Hold up! Look, the problem is that my grant was not permitted to the extent that I asked for. Those idiots just don't understand the relevance of my research and cut my funds. Only because I'm still a student they completely underestimate my abilities -- by far!
+    Now I spent all my credits on this sensor suit without the possibility to use it. You must know how this feels like. I mean, your ship is obviously in a very bad shape. So why don't you just help me out here?"]])
+text[3] = _([["You do not have enough free cargo space to accept this mission!"]])
+text[4] = _([["Great! I'll start to load the sensors into your ship right away. We should be ready to take off soon."]])
+text[5] = _([["So it is not a problem at all? I'm still a student and spent all funds I got on the sensor suit. Thank you for helping me out here! I'll start to load the sensors into your ship right away. We should be ready to take off soon."]])
+text[6] = _([[As you enter your ship you notice dozens of cables of various colors stretched across your ship's corridors. It is a complete mess. Even weirder is the fact that it is less than one period ago that you met the student in the bar. What has he done to your ship?
+    You follow the direction where most of the cable seem to lead to. Finally you find the culprit.]])
+text[7] = _([["Oh, hello again, Captain! I'm done just by now here, so we can take off whenever you're ready. I have to calibrate the sensors during the flight so there is no need to rush. Our first destination is %s."
+    After asking what he has done to your ship he just answers, "Why, I just installed the sensors as appointed. It should have no unwanted side effects on your ship."]])
+text[8] = _([["Just a complete mess? Haven't you noticed the color coding? Well, I have my own system so it may appears confusing to you. Don't worry, I know exactly what I'm doing!"
+    His last words are supposed to be reassuring but instead you realize accepting this mission was not the best idea...]])
```

Comment:
Haven't verified this, but it seems like you're breaking up text[6], text[7], and text[8] a bit more than is necessary here. Actually it sort of looks like all three of these should be one message box.

I would also note that these paragraph breaks in all three of these seem unnecessary. This looks to me like it should be a single string  consisting of three paragraphs, or if it's big enough to cause size concerns, two paragraphs in a first one and one in a second one. Though that's an English style thing and easy to fix, so I'm not too worried about that.

🕵️